docs/src/gui/qtvcp-code-snippets.adoc

```diff
@@ -1,18 +1,22 @@
 :lang: en
 :toc:
 
-[[cha:qtvcp-code]]
-= QtVCP Handler file code snippets
+[[cha:qtvcp:code]]
+= QtVCP Handler File Code Snippets
 
 Here are bits of ideas to put in the handler file.
```

Comment:
This looks a bit lost. Can this be removed?

Reply:
I like a consistent formatting of achors, like [document:header] across all documents. And the start with a capital letter in the top-level heading seems also ok?

Reply:
This comment should be linked to line 7, which isn't obvious as I just see :D

It's about this line appearing before any section:
"Here are bits of ideas to put in the handler file."